Source/Urho3D/Scene/Serializable.h

```diff
@@ -87,6 +87,11 @@ class URHO3D_API Serializable : public Object
     virtual bool LoadJSON(const ea::string& resourceName);
     /// Load from resource of automatically detected type.
     virtual bool LoadFile(const ea::string& resourceName);
+    /// Save with auto detect type. with "RootNode" default name
+    bool SaveFile(const ea::string& destinationPath);
+
```

Comment:
No need for new line.

Reply:
omitted for now dues to unexpected results